docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-creating-a-case-proc.adoc

```diff
@@ -1,3 +1,32 @@
 [id='case-management-creating-a-case-proc']
 = Creating a case in {PRODUCT}
 
+A case project is a type of project in {CENTRAL} that is configured for case management. When you create a new case project, the following configurations are set automatically in {CENTRAL}:
+
+* Runtime strategy is set to Per Case
+* Case and documents marshallers are configured.
```

Comment:
Similar to the above bullet comment. A little context about what and why Case and document marshallers are would help. There is some helpful context in the 3rd bullet. 

Don't hate me